_posts/2018-11-16-stairway-to-health.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+authors: [gert-jan_schokkaert]
+title: "Visualising IoT data with tableau"
+image: /img/2018-11-16-stairway-to-health/stairway-to-health.png
+tags: [Development,Translations,Angular,i18n,Crowdin]
+category: Development
+comments: true
+---
+
+# VisionWorks meets JWorks: StairwayToHealth
+We from [VisionWorks][] were asked to rebuild the visualisation dashboard [JWorks][] used in the [application][] they built as a result of the internal StairwayToHealth project (you can find more information about that project [here][]). We decided to use [Tableau][], a popular BI Visualisation tool we largely use at our clients.
+We developed the dashboard working around some key questions while keeping the appearance of the dashboard in line with the dashboard JWorks developed.
+
+In the following section we will explain how the dashboard is currently set up and how to use it properly.
+Next we will go over the features we can add in future releases to allow the user to go even deeper in their analysis.
+
+# Dashboard overview
+
+The dashboard is built to answer **the following questions**:
+
+1. What is the percentage of people taking the stair/elevator at Ordina **today**?
```

Comment:
Any reason why "stair" is used everywhere instead of stairs? I'd personally use "stairs" everywhere.